Tests/test_main.py

```diff
@@ -5,8 +5,16 @@
 import sys
 from unittest import TestCase
 
+from .helper import on_github_actions, unittest
+
 
 class TestMain(TestCase):
+    @unittest.skipIf(
+        sys.platform == "win32"
+        and hasattr(sys, "pypy_translation_info")
```

Comment:
Minor thing, there's an `is_pypy()` in `test_core_resources.py` that could be moved to `helper.py`

Reply:
How about moving `sys.platform == "win32"` to `helper.is_win32()` as well? A quick grep found 20 matches in `Tests`.

Reply:
Sure! There's also a `skipKnownBadTest` in `helper.py`. This could be done in another PR if you prefer.

Reply:
`is_pypy` and `is_win32` are done, let's do `skipKnowBadTest` in a future PR.